examples/amp-story/interactive_results_detailed.html

```diff
@@ -339,25 +270,93 @@ <h1>Detailed Results Component</h1>
+            option-4-results-category="National Park lover"
           >
           </amp-story-interactive-results-detailed>
         </amp-story-grid-layer>
       </amp-story-page>
     </amp-story>
+    <script>
+      document.getElementById('add-quizzes').onclick = function (e) {
+        var quizCounter = document.getElementById('quiz-count');
+        var quizzes = document.getElementById('quizzes');
+        for (var i = 0; i < quizCounter.value; i++) {
+          var quiz = document.createElement('amp-story-interactive-quiz');
+          quiz.setAttribute('id', 'quiz-' + i);
+          quiz.setAttribute('endpoint', 'https://webstoriesinteractivity-beta.web.app/api/v1');
+          quiz.setAttribute('option-1-text', 'Statue of Liberty');
+          quiz.setAttribute('option-1-correct', '');
+          quiz.setAttribute('option-2-text', 'Golden Gate Bridge');
+          quizzes.append(quiz);
+        }
+      }
+
+      document.getElementById('add-img-quizzes').onclick = function () {
+        var imgQuizCounter = document.getElementById('img-quiz-count');
+        var imgQuizzes = document.getElementById('img-quizzes');
+        for (var i = 0; i < imgQuizCounter.value; i++) {
+          var imgQuiz = document.createElement('amp-story-interactive-img-quiz');
+          imgQuiz.setAttribute('id', 'img-quiz-' + i);
+          imgQuiz.setAttribute('endpoint', 'https://webstoriesinteractivity-beta.web.app/api/v1');
+          imgQuiz.setAttribute('option-1-image', 'https://images.unsplash.com/photo-1565475783696-96001eff1b45?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1500&q=80');
+          imgQuiz.setAttribute('option-1-image-alt', 'Statue of Liberty');
+          imgQuiz.setAttribute('option-1-correct', '');
+          imgQuiz.setAttribute('option-2-image', 'https://images.unsplash.com/photo-1610312278520-bcc893a3ff1d?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1460&q=80');
+          imgQuiz.setAttribute('option-2-image-alt', 'Golden Gate Bridge');
+          imgQuizzes.append(imgQuiz);
+        }
+      }
+
+      document.getElementById('add-polls').onclick = function () {
+        var pollCounter = document.getElementById('poll-count');
+        var polls = document.getElementById('polls');
+        for (var i = 0; i < pollCounter.value; i++) {
+          var poll = document.createElement('amp-story-interactive-poll');
+          poll.setAttribute('id', 'poll-' + i);
+          poll.setAttribute('endpoint', 'https://webstoriesinteractivity-beta.web.app/api/v1');
+          poll.setAttribute('option-1-text', 'Lake Tahoe');
+          poll.setAttribute('option-1-results-category', 'Lake lover');
+          poll.setAttribute('option-2-text', 'Hong Kong');
+          poll.setAttribute('option-2-results-category', 'City-goer');
+          polls.append(poll);
+        }
+      }
+
+      document.getElementById('add-img-polls').onclick = function () {
```

Comment:
This logic is very clever but tying these elements to the pages makes the story behave weirdly. Eg: sliding on the range will advance the page. Sth that would make more sense is:
- Add a sibling of amp-story that's positioned in the top left corner
- Contain 4 input ranges that go from 0-5 and set the initial value to 2-3 (the story should start with some components on each page)
- Listen for `change` events on the input ranges and update the number of components. No need for an extra button.

This way, you can use an overlay to modify the story, but by default you can test it. There's a few places where we do this (add an overlay near a corner to modify the story)